embc-app/ClientApp/src/app/shared/components/date-time-picker/date-time-picker.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="row">
+  <section class="col">
+    <label class="small" for="startDate">Date</label>
+  </section>
+  <section class="col">
+    <label class="small" for="startTime">Time</label>
+  </section>
+</div>
+<div class="row">
+  <section class="col vertical-spacer">
+    <input type="text" class="form-control" [value]="dateString" (change)="updateDate($event.target.value)" placeholder="yyyy-mm-dd" id="startDate" autocomplete="off">
```

Comment:
Should this be a date picker?

Reply:
This could be addressed in a future enhancement